### PR TITLE
Increase the number of injected packets to pass the packet sampling t…

### PR DESCRIPTION
**Pull request by fredyu190011**
### Description of PR

Increase the number of injected packets to pass the packet sampling test case

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [X] Test case(new/improvement)

#### What is the motivation for this PR?

By current verified, the models(4/5/6/7/9) series using Broadcom chip, injected 50*sample_rate packets is not enough to pass the packet sampling test case.

The value needs to increase from 50*sample_rate to 100*sample_rate.

#### How did you do it?

Increase the packets that ptf will send. We adjust the number from 50*sample_rate to 100*sample_rate.

#### How did you verify/test it?

Run all packet sampling relating test cases on AS5835-54X on t0 topology.
The result shown all are passed. The test cases are listed below.

sflow/test_sflow.py::TestSflowCollector::test_sflow_config
sflow/test_sflow.py::TestSflowCollector::test_collector_del_add
sflow/test_sflow.py::TestSflowCollector::test_two_collectors
sflow/test_sflow.py::TestSflowInterface::testIntfRemoval
sflow/test_sflow.py::TestSflowInterface::testIntfSamplingRate
sflow/test_sflow.py::TestSflowInterface::testIntfChangeSamplingRate

### Details if related

Signed-off-by: Fred Yu fred_yu@edge-core.com

### CODE REVIEW

File: ansible/roles/test/files/ptftests/sflow_test.py [hunk 1]
```diff
@@ -193,14 +193,14 @@ def analyze_counter_sample(self, data, collector, polling_int, port_sample):
 
     def analyze_flow_sample(self, data, collector): 
         logging.info("packets collected from interfaces ifindex : %s" %data['flow_port_count'])
-        logging.info("Expected number of packets from each port : %s to %s"%(50*0.6,50*1.4))
+        logging.info("Expected number of packets from each port : %s to %s"%(100*0.6,100*1.4))
```
Comment (wangxin) [inline ansible/roles/test/files/ptftests/sflow_test.py:196]: Can you add spacing in between for better readability? For example:
`(100*0.6,100*1.4)` -> `(100 * 0.6, 100 * 1.4)`

File: ansible/roles/test/files/ptftests/sflow_test.py [hunk 2]
```diff
@@ -193,14 +193,14 @@ def analyze_counter_sample(self, data, collector, polling_int, port_sample):
...
         for port in self.interfaces:
             ifindex = self.interfaces[port]['ifindex'] 
             logging.info("....%s : Flow packets collected from port %s = %s"%(collector,port,data['flow_port_count'][ifindex]))
             if port in self.enabled_intf :
                 # Checking samples with tolerance of 40 % as the sampling is random and  not deterministic.Over many samples it should converge to a mean of 1:N
-                # Number of packets sent = 50 * sampling rate of interface 
-                self.assertTrue(50*0.6 <= data['flow_port_count'][ifindex] <= 50*1.4 ,
+                # Number of packets sent = 100 * sampling rate of interface
+                self.assertTrue(100*0.6 <= data['flow_port_count'][ifindex] <= 100*1.4 ,
```
Comment (wangxin) [inline ansible/roles/test/files/ptftests/sflow_test.py:203]: Can you add spacing in between for better readability?

File: ansible/roles/test/files/ptftests/sflow_test.py [hunk 3]
```diff
@@ -213,8 +213,8 @@ def sendTraffic(self):
         ip_dst_addr = '192.168.0.4'
         src_mac = self.dataplane.get_mac(0, 0)
         pktlen=100
-        #send 50*sampling_rate packets in each interface for better  analysis
-        for j in range(0,50,1):
+        #send 100*sampling_rate packets in each interface for better  analysis
+        for j in range(0,100,1):
```
Comment (wangxin) [inline ansible/roles/test/files/ptftests/sflow_test.py:217]: Can you add spacing in between for better readability?

File: ansible/roles/test/files/ptftests/sflow_test.py (hunk 2 above)
Comment (fredyu190011) [reply ansible/roles/test/files/ptftests/sflow_test.py]: Done.

File: ansible/roles/test/files/ptftests/sflow_test.py (hunk 3 above)
Comment (fredyu190011) [reply ansible/roles/test/files/ptftests/sflow_test.py]: Done.

File: ansible/roles/test/files/ptftests/sflow_test.py (hunk 1 above)
Comment (fredyu190011) [reply ansible/roles/test/files/ptftests/sflow_test.py]: Done.

**Review by wangxin — APPROVED** [review]

**Comment by wangxin** [comment]
retest vsimage please

**Pull request merged**